backend/lib/spree/backend_configuration.rb

```diff
@@ -6,6 +6,22 @@ module Spree
   class BackendConfiguration < Preferences::Configuration
     preference :locale, :string, default: I18n.default_locale
 
+    # @!attribute [rw] themes
+    #   @return [Hash] A hash containing the themes that are available for the admin panel
+    preference :themes, :hash, default: {
+      classic: 'spree/backend/all',
+    }
+
+    FALLBACK_THEME = :classic
```

Comment:
I think we forgot to use the constant, as other references use `:classic` every time :slightly_smiling_face: